<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="human_tf" value="human" />
    <param name="action_name" type="string" value="/move_to_human" />
    <param name="move_base_plan_service_name" type="string" value="/move_base/make_plan" />
    <param name="move_base_action_name" type="string" value="/move_base" />
    <param name="move_head_action_name" type="string" value="/head_controller/point_head_action" />
    <param name="odometry_topic" type="string" value="/mobile_base_controller/odom" />
    <param name="human_tf" type="string" value="$(arg human_tf)" />
    <param name="base_link" type="string" value="base_link" />
    <param name="head_turning_velocity" type="double" value="0.4" />
    <param name="min_distance_to_human" type="double" value="1.0" />

    <node pkg="dialogflow_emergency_action_servers" name="move_to_human_action_server" type="move_to_human_action_server.py" output="screen" />
</launch>
